tooling/scripts/generate_matrix.sh

```
@@ -118,15 +123,16 @@ function get_workflow() {
 
 function generate_matrix_object() {
   local projects="$1"
+  local root_path="$2"
   local matrix_object="{\"include\":["
```

Shellcheck (0.10.0)

[warning] Quotes/backslashes will be treated literally. Use an array.

(SC2089)